pkgs/development/tools/jf/default.nix

```diff
@@ -1,17 +1,28 @@
-{ lib, stdenv, rustPlatform, fetchFromGitHub }:
+{ lib, stdenv, rustPlatform, fetchFromGitHub, installShellFiles }:
 
 rustPlatform.buildRustPackage rec {
   pname = "jf";
-  version = "0.2.2";
+  version = "0.3.3";
 
   src = fetchFromGitHub {
     owner = "sayanarijit";
     repo = "jf";
     rev = "v${version}";
-    hash = "sha256-6x/WDsDHkQVjiere17XMY3lxG2M5bmTtdH1rMjii+NY=";
+    hash = "sha256-dZX8C/nJaKfgHsUAXR1DRZS+PWDZF+QDVpOSaOlwFp4=";
   };
 
-  cargoHash = "sha256-W8/6EiqevF7mX2cgdv9/USVnSVxQ/J6I3Cq/UJyZOxU=";
+  cargoHash = "sha256-H9UZCKy+0xL6J8f/6yCiM4X5TMOrN8UEEDwxqNR7xQY=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  # skip auto manpage update
+  checkFlags = [
+    "--skip tests::update_manpage"
```

Comment:
```suggestion
    "--skip=tests::update_manpage"
```

Reply:
alternatively you can use `buildNoDefaultFeatures = true;`, which should disable the manpage feature and skip this test